pkg/storage/storagebackend/factory/factory.go

```diff
@@ -26,9 +26,9 @@ import (
 // Create creates a storage backend based on given config.
 func Create(c storagebackend.Config) (storage.Interface, error) {
 	switch c.Type {
-	case storagebackend.StorageTypeUnset, storagebackend.StorageTypeETCD2:
+	case storagebackend.StorageTypeETCD2:
 		return newETCD2Storage(c)
-	case storagebackend.StorageTypeETCD3:
+	case storagebackend.StorageTypeUnset, storagebackend.StorageTypeETCD3:
```

Comment:
As long as I'm probably fine with changing the default for tests in this PR, I don't think we should do it for binaries (since we still don't have rollback etc., so we may not able to launch it).